app/models/restart_signal_handler.rb

```diff
@@ -25,15 +25,40 @@ def run
     JobExecution.enabled = false
 
     until JobExecution.active.empty? && MultiLock.locks.empty?
-      puts "Waiting for jobs: #{JobExecution.active.map(&:id)}"
+      loginfo = {
+        jobs: JobExecution.active.map do |job_exec|
+          {
+            job_id: job_exec.id,
+            pid: job_exec.pid,
+            pgid: job_exec.pgid,
+            project: job_exec.job.project.name
+          }
+        end,
+        locks: MultiLock.locks
```

Comment:
As far as I can tell, both the key and the value of `MultiLock.locks` are strings which contain useful information, so I figure I'd just output them both.